File: examples-sdk/zig/018-basic-ai-multi-model-router/main.zig
// 018-basic-ai-multi-model-router — Zig SDK equivalent
// Compile: vil compile --from zig --input 018-basic-ai-multi-model-router/main.zig --release

const vil = @import("vil");

pub fn main() void {
    var server = vil.Server.init("ai-multi-model-router", 3085);
    var router = vil.Service.init("router");
    router.endpoint("POST", "/route", "route_handler");
    server.service(&router);
    server.compile();
}

File: examples-sdk/go/018-basic-ai-multi-model-router/main.go
// 018-basic-ai-multi-model-router — Go SDK equivalent
// Compile: vil compile --from go --input 018-basic-ai-multi-model-router/main.go --release
package main

import vil "github.com/OceanOS-id/vil-go"

func main() {
	s := vil.NewServer("ai-multi-model-router", 3085)

	router := vil.NewService("router")
	router.Endpoint("POST", "/route", "route_handler")
	s.Service(router)

	s.Compile()
}
